packages/opc-styles/functions/_tools.scss

```diff
@@ -0,0 +1,23 @@
+// https://hugogiraudel.com/2013/08/08/advanced-sass-list-functions
+/// Return `$list` as a string connected by `$glue`
+/// @author Hugo Giraudel
```

Comment:
are u crediting the original author here??

Reply:
yes.